ci/setup-macos.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+set -x
+
+brew update
+brew install pkgconfig
+brew install zlib
+brew install curl
+brew install openssl
+brew install libssh2
```

Comment:
There's no need to list them all separately, you can just pass multiple formulas to brew (even though it is not documented).